wave_matching/src/ndt.cpp

```diff
@@ -0,0 +1,63 @@
+#include "wave/utils/config.hpp"
+#include "wave/utils/logging.hpp"
+#include "wave/matching/ndt.hpp"
+
+namespace wave {
+
+NDTMatcher::NDTMatcher(float res, const std::string &config_path) {
```

Comment:
same as the ICP comment, make constructor that takes in a subset of the most important params, default the rest.  That way if we want multiple matchers with different params (think multi-scale optimization), we don't need a bunch of different config files, or have to call set methods all the time